controllers/kernelchaos/types.go

```diff
@@ -17,6 +17,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/go-multierror"
```

Comment:
group imports

Reply:
It was inserted by GoLand automatically. It seems it is not smart enough to group imports.